test/agent.js

```diff
@@ -18,6 +18,13 @@ var inContainer = 'containerId' in (containerInfo() || {})
 process.env.ELASTIC_APM_METRICS_INTERVAL = '0'
 process.env.ELASTIC_APM_CENTRAL_CONFIG = 'false'
 
+test('#getServiceName()', function (t) {
+  var agent = Agent()
+  agent.start()
```

Comment:
nit: since this is a public API and users are going to user, we could also capture how we expect `getServiceName` to behave prior to `start`being called.

Reply:
I've updated tests and docs.